cmd/podman/root.go

```diff
@@ -338,6 +338,10 @@ func rootFlags(cmd *cobra.Command, opts *entities.PodmanConfig) {
 
 	lFlags := cmd.Flags()
 
+	sshFlagName := "ssh"
+	lFlags.StringVar(&opts.SSHMode, sshFlagName, "golang", "define the ssh mode")
```

Comment:
```suggestion
	lFlags.StringVar(&opts.SSHMode, sshFlagName, ssh.GolangMode "define the ssh mode")
```

I think there's a const in the `ssh` package for it.